packages/services/api/src/modules/support/providers/support-manager.ts

```diff
@@ -606,6 +622,9 @@ export class SupportManager {
       metadata: {
         ticketDescription: input.description,
         ticketPriority: input.priority,
+        ticketCategory: input.category ?? '',
+        ticketProject: input.project ?? '',
+        ticketTarget: input.target ?? '',
```

Comment:
prefer not including the metadata if it isnt present here.
I.e. `...(input.category ? { ticketCategory: input.category } : {}),`